Writerside/topics/project-management/code-style-guide.md

```diff
@@ -80,7 +80,98 @@ public class ExampleServiceTest {
 
 ## C++
 
-TODO...
+Following section outlines the C++ coding guideline focusing on formatting, naming conventions, and testing
+practices. The code example demonstrates how to implement and test a simple Example class.
+
+**Formatting**
+Always adhere to the project's defined code formatting. You should use the .clang-format file to
+enforce consistent styling across the codebase.
+
+- Use the formatter defined in the .clang-format file.
+    - example
```

Comment:
Is this an example or the code style guide to actually use in the project?

Reply:
@munterfi it is just an example of how the formatting yml file looks. 
If unnecessary, we can delete it. I have added it so that reader can get an idea of the clang-format file